openmc/settings.py

```diff
@@ -945,7 +946,18 @@ def weight_windows_on(self) -> bool:
     def weight_windows_on(self, value: bool):
         cv.check_type('weight windows on', value, bool)
         self._weight_windows_on = value
-
+    
+    @property
+    def weight_window_checkpoints(self) -> dict:
+        return self._weight_window_checkpoints
+    
+    @weight_window_checkpoints.setter
+    def weight_window_checkpoints(self, weight_window_checkpoints: dict):
+        for key in weight_window_checkpoints.keys():
+            if key not in ('collision', 'surface'):
+                raise ValueError('Wrong values on weight_window_checkpoints. Use either \"collision\" or \"surface\".')
```

Comment:
@XinyanBradley I should've suggested this the first time. For consistency we should probably use the `check_value` module.
```suggestion
            cv.check_value('weight_window_checkpoint', key, 
            ('collision', 'surface'))
```

Reply:
I will address this in the next commit.

Reply:
Addressed in the latest commit